src/build/__init__.py

```diff
@@ -201,15 +187,21 @@ def check_dependencies(self, distribution):  # type: (str) -> Set[str]
 
         return {dep for dep in dependencies if not check_version(dep)}
 
-    def build(self, distribution, outdir):  # type: (str, str) -> None
+    def build(self, distribution, outdir=None):  # type: (str, Optional[str]) -> None
         """
         Builds a distribution
 
         :param distribution: Distribution to build (sdist or wheel)
         :param outdir: Output directory
         """
         build = getattr(self.hook, 'build_{}'.format(distribution))
-        outdir = os.path.abspath(outdir)
+
+        if outdir is None:
+            # Resolve outdir relative to project root if not specified.
```

Comment:
IMHO the link to the issue is not needed, and the comment can start with lowercase letter, and end without the period.